### ISSUE TITLE
Update logic in common run scripts to get date info from set_env.ksh and update namelists.

### ISSUE THREAD

**Issue by JohnHalleyGotway** [NCAR/container-dtc-nwp#9]
Modify the scripting logic to make running new cases easier.
Currently, you have to update the begin/end dates in several places: namelist.input, namelist.wps, and 2 spots in set_env.ksh.
Update the run scripts (run_wps.ksh and run_real.ksh) to source set_env.ksh and modify the namelists.  Also, update the run_met.ksh so that it uses the date info that's set for UPP.

**Comment by JohnHalleyGotway** [comment]
We discussed this on 9/14/21 and @fossell will write up a new issue for standardizing the variable names used in set_env.ksh. For example, the WRF start time is listed at least 4 times. These changes should be done prior to any work on this issue.

While this issue specifically mentions date information, be sure to check for other variables that are set in multiple spots and consider defining them once in set_env.ksh instead. And if those variables exist, recommend renaming this issue in a more generic way.
